functions/package.json

```diff
@@ -3,8 +3,8 @@
   "description": "Cloud Functions for Firebase",
   "packageManager": "yarn@1.22.22",
   "scripts": {
-    "lint": "eslint --ext .js,.ts .",
-    "build": "tsc",
+    "lint": "eslint ./src --report-unused-disable-directives",
```

Comment:
These changes are not correct. We should not change the existing lint and build scripts. We only want to add the checks on CI